Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -65,11 +65,15 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.tableView.accessibilityIdentifier = "Bookmarks List"
+        self.setupBookmarkSource()
```

Comment:
Some questions:

1. Why do we need to call `setupBookmarkSource` in both `viewDidLoad` and `viewWillAppear`? You said `viewWillAppear` isn't always called, but I think `viewDidLoad` *is* always called, so I don't see why we'd need to call this in two places.

1. When/why isn't `viewWillAppear` called? I wonder if [this](http://stackoverflow.com/a/39161026/1524401) is related.

1. Why were we using `viewWillAppear` in the first place?